Signal/src/environment/PropertyListPreferences.m

```diff
@@ -130,9 +120,21 @@ - (void)setHasRegisteredVOIPPush:(BOOL)enabled
     [self setValueForKey:PropertyListPreferencesKeyHasRegisteredVoipPush toValue:@(enabled)];
 }
 
+- (BOOL)loggingIsEnabled
+{
+    NSNumber *preference = [NSUserDefaults.standardUserDefaults objectForKey:PropertyListPreferencesKeyEnableDebugLog];
```

Comment:
This looks good, but it doesn't look like it will migrate the old preferences value. When we load the database, I proposed that we:

* Check the old preference. If it's set to "disable logging", then:
* Remove the old preference if possible or set it to "enable logging" if not (to avoid overwriting the new property twice) and:
* Set new new preference to "disable logging".